CoreTableView/nbproject/project.xml

```diff
@@ -14,6 +14,14 @@
                         <specification-version>1.0</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
```

Comment:
...and don't forget to remove the dependency on Core Functionality here.

Reply:
Done.